### PR TITLE
PostPicker - Implement replace and reset buttons

### PR DESCRIPTION
**Pull request by ellm**
https://github.com/alleyinteractive/alley-scripts/issues/153

### CODE REVIEW

File: packages/block-editor-tools/src/components/post-picker/index.tsx [hunk 1]
```diff
@@ -98,6 +90,29 @@ const PostPicker = ({
     setShowModal(false);
   };
 
+  const controls = () => (
+    <ButtonGroup>
+      <Button
+        variant="primary"
```
Comment (mogmarsh) [inline packages/block-editor-tools/src/components/post-picker/index.tsx:96]:
```suggestion
        variant="primary"
```
I feel like with 2 buttons, they shouldn't both be primary. One could be primary and one secondary, or both secondary. Could also experiment with `isDestructive` on the Reset button.

File: packages/block-editor-tools/src/components/post-picker/index.tsx [hunk 2]
```diff
@@ -98,6 +90,29 @@ const PostPicker = ({
...
+        onClick={onReset}
+        style={{
+          margin: '0 4px',
+        }}
+      >
+        {__('Reset', 'alley-scripts')}
+      </Button>
+      <Button
+        variant="primary"
```
Comment (mogmarsh) [inline packages/block-editor-tools/src/components/post-picker/index.tsx:105]:
```suggestion
        variant="secondary"
```

**Review by mogmarsh — APPROVED** [review]
thanks for taking this on! I haven't had a chance to get back to it.
We should also bump the version to `0.3.1` [here](https://github.com/alleyinteractive/alley-scripts/blob/main/packages/block-editor-tools/package.json#L3) 👍

File: packages/block-editor-tools/src/components/post-picker/index.tsx (hunk 1 above)
Comment (ellm) [reply packages/block-editor-tools/src/components/post-picker/index.tsx:96]: Good call! Yeah, that makes sense.

**Comment by ellm** [comment]
Thanks @mogmarsh ! I'll leave this open for you or another admin to merge. I'll try to help with other open issues, if I can.

**Pull request merged**